ffi/capi/src/collator.rs

```diff
@@ -158,7 +158,7 @@ pub mod ffi {
         /// and the options from locale data were combined. None of the struct fields
         /// will have `Auto` as the value.
         #[diplomat::rust_link(icu::collator::Collator::resolved_options, FnInStruct)]
-        #[diplomat::attr(supports = accessors, getter)]
+        #[diplomat::attr(*, getter)]
```

Comment:
thought: we can also have it be such that having a single attribute means no cfg